aiida/transports/plugins/ssh.py

```diff
@@ -533,6 +545,22 @@ def normalize(self, path='.'):
         """
         return self.sftp.normalize(path)
 
+    def stat(self, path):
+        """
+        Wrap SFTP stat call
```

Comment:
It would be more useful to know exactly what this method is supposed to do and what it returns. The fact that it wraps `sftp.stat` in this implementation is not that useful. Since you had to implement this in pure ssh I think you know what it should return. Could you please add that in the docstring instead? Same for `lstat`.

Reply:
I added some comments for these calls in  88b88c9